src/libs/OptionsListUtils.js

```diff
@@ -204,20 +204,41 @@ function hasReportDraftComment(report) {
         && lodashGet(reportsWithDraft, `${ONYXKEYS.COLLECTION.REPORTS_WITH_DRAFT}${report.reportID}`, false);
 }
 
+/**
+ * @param {Object} report
+ * @returns {Array<String>}
+ */
+function getParticipants(report) {
+    const participants = [...lodashGet(report, ['participants'], [])];
```

Comment:
Let's test it? 90% sure can be 100% wrong :trollface:

Reply:
Ah yeah, I added on purpose because otherwise we mutate `report.participants` and we end up with tons of repeated participants when we add the admin and/or report owner a few lines below.